application/common/util-native/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+import tools.forma.android.config.NdkAbi
+import tools.forma.android.config.CMake
+
+androidNative(
+    packageName = "com.stepango.blockme.common.util.native",
+    owner = Teams.core,
+    abi = setOf(NdkAbi.ARM7, NdkAbi.ARM8, NdkAbi.X86),
+    buildSystem = CMake(
+        path = File(projectDir, "src/main/jni/CMakeLists.txt"),
+        version = "3.10.2"
```

Comment:
Define as a global version through formaConfiguration?

Reply:
I don't really know, is it possible to specify several versions of CMake within single target in the old versions of android plugin, but i'm working on it.